org.talend.mdm.core/src/com/amalto/core/delegator/ISecurityCheck.java

```
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2006-2018 Talend Inc. - www.talend.com
+ * 
+ * This source code is available under agreement available at
+ * %InstallDIR%\features\org.talend.rcp.branding.%PRODUCTNAME%\%PRODUCTNAME%license.txt
+ * 
+ * You should have received a copy of the agreement along with this program; if not, write to Talend SA 9 rue Pages
+ * 92150 Suresnes, France
+ */
+package com.amalto.core.delegator;
+
+import com.amalto.core.save.DocumentSaverContext;
+import com.amalto.core.save.SaverSession;
+
+public class ISecurityCheck {
+
+    public boolean isByPassSecurityChecks(SaverSession session, DocumentSaverContext context) {
```

Wouldn't it be better to throw an Exception  and name the method differently such

`public void vetoableSave(SaverSession session, DocumentSaverContext context) throws VetoException`

---

Changed accordingly, please have a look.